src/components/molecules/Settings/SettingsHeader/index.tsx

```diff
@@ -20,12 +20,12 @@ const SettingsHeader: React.FC<Props> = ({ title, currentWorkspace, currentProje
 
   return (
     <Wrapper>
-      <Text size="xl" color={theme.main.strongText} weight="bold">
+      <Text size="xl" color={theme.main.strongText} weight="normal">
         {workspace} {workspace && (title || currentProject) && " / "}
         {currentProject} {title && currentProject && " / "}
         {title}
       </Text>
-      <Text size="m" otherProperties={{ marginTop: "12px" }}>
+      <Text size="m" className={theme.main.text} otherProperties={{ marginTop: "12px" }}>
```

Comment:
Did you mean this className to be color??

Reply:
oh, yes, thanks